src/main/java/com/internet/shop/dao/OrderDaoImpl.java

```diff
@@ -0,0 +1,34 @@
+package com.internet.shop.dao;
+
+import com.internet.shop.lib.Dao;
+import com.internet.shop.models.Order;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class OrderDaoImpl implements OrderDao {
```

Comment:
in this PR we should only have ProductDao an ProductService

Reply:
deleted